### DISCUSSION TITLE
Task Graph not working for any vehicles other than LTV1

### DISCUSSION THREAD

**Discussion by jbain1029** [SimDynamX/SpaceTeamsPRO#71] [2024 Lunar SAR Competition Q&A]
Task Graph for Scouts and LTV2 aren't working beyond the first task in the Task Graph. I have verified that the task graphs contain all the tasks they should, and I have also tried the example code provided in the competition files, but none of them work except for the first task. I have not changed anything in TaskGraph.py either.

**Comment by connorjak** [comment]
Should be fixed once you've made oncommandcomplete reactions for all the commands you use per entity. STU Lunar SAR v1.2.1 has fixes which make the warnings for missing the reaction actually show up, among other changes.

**Comment by connorjak** [comment]
Also with v1.2.1, the SetupAllCommands function needs to be called for each entity before any of the commands will work. If that is missing, none of the commands will complete.
